### PR TITLE
Ph 1083 empty search request

### PR DESCRIPTION
**Pull request by rimvydas-hashtages**

### CODE REVIEW

**Comment by rimvydas-hashtages** [comment]
@simonhintersonnleitner build seems failing because it cant pull plenty-ceres repo `Error: Failed to clone https://github.com/plentymarkets/plugin-ceres via https, ssh protocols, aborting.` Maybe we should try upping the plenty lts version

**Comment by Bibarella** [comment]
> @simonhintersonnleitner build seems failing because it cant pull plenty-ceres repo `Error: Failed to clone https://github.com/plentymarkets/plugin-ceres via https, ssh protocols, aborting.` Maybe we should try upping the plenty lts version

Ran it again and it worked

File: resources/js/src/app/shared/UrlBuilder.ts [hunk 1]
```diff
@@ -16,12 +16,22 @@ class UrlBuilder {
...
-            params[decodeURIComponent(tokens[1])] = decodeURIComponent(tokens[2]);
+            params[decodeURIComponent(tokens[1])] = this.sanitizeHTML(decodeURIComponent(tokens[2]));
         }
 
         return params;
     }
 
+    sanitizeHTML(str : string): string {
+
+        const map = {
```
Comment (Bibarella) [inline resources/js/src/app/shared/UrlBuilder.ts:27]: Extend map to handle additional characters, such as > (&gt;) and & (&amp;) please

**Review by Bibarella — CHANGES_REQUESTED** [review]

**Review by simonhintersonnleitner — COMMENTED** [review]
⚙️  Please keep in this MR just the changes the needed for the fix. It' looks like that there are change from the other [MR](https://github.com/findologic/plugin-plentymarkets-ceres-api/pull/386).

**Pull request closed without merging**